web3/middleware/stalecheck.py

```diff
@@ -26,7 +26,10 @@
 
 
 def _isfresh(block: BlockData, allowable_delay: int) -> bool:
-    return block and time.time() - block['timestamp'] <= allowable_delay
```

Comment:
mypy didnt like this with parens?

Reply:
I suspect the problem was the `return block` because it was complaining that the return value could either be `BlockData` or a `bool`, but we only specify bool. I don't love the explicit `return True/False` but it is explicit and it makes mypy happy 🤷